README.md

```diff
@@ -20,10 +20,17 @@
 ember install ember-frost-sidebar
 ```
 ## API
-Coming soon
+Not applicable at the moment.
 
 ## Examples
-Coming soon
+```
+{{#frost-sidebar class="demo-instance"}}
+  <div class="demo-pod-content">
+    <h4>Test content</h4>
+    {{frost-checkbox  checked=true label="Check me"}}
+  </div>
+{{/frost-sidebar}}
+```
 
 ## Development
```

Comment:
Actually, I think it is good - I thought the text was not there as the PR did not seem to have any content.  I checked the full file and it looks good.  Sorry about it.